testsuite/features/secondary/srv_cobbler_buildiso.feature

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+@scope_cobbler @scope_iso
+Feature: Cobbler buildiso
+
+  Scenario: Log in as testing user in the cobbler buildiso context
+    Given I am authorized as "testing" with password "testing"
+
+  Scenario: Copy cobbler profiles on the server in the cobbler buildiso context
+    When I copy autoinstall mocked files on server
+
+  Scenario: Create a dummy distro in the cobbler buildiso context
+    Given cobblerd is running
+    Then create distro "testdistro" as user "testing" with password "testing"
+
+  Scenario: Create dummy profiles in the cobbler buildiso context
+    Given cobblerd is running
+    And distro "testdistro" exists
+    Then create profile "orchid" for distro "testdistro" as user "testing" with password "testing"
+    Then create profile "flame" for distro "testdistro" as user "testing" with password "testing"
```

Comment:
I would substitute these redundant `Then`s with `And`s, after the first `Then` so it's easier to read and more consistent with the rest of the testsuite :thinking:

Reply:
Fixed.